staker/staker.gno

```diff
@@ -43,45 +43,52 @@ func Init(token grc20.Token) {
 }
 
 // StakeAmount stakes the specified amount for the given delegate
-func StakeAmount(delegate std.Address, amount *u256.Uint) error {
+func StakeAmount(delegate string, amount uint64) error {
+	delegateAddr := std.Address(delegate)
```

Comment:
Check if delegate is valid address

Reply:
I'm not sure how to check valid address propoerly, So I just comparing is this empty string or not.

https://github.com/gnoswap-labs/governance/pull/14/commits/37845cc98a4cf14e140259bd291469192121dce6

Reply:
gno has IsValid for address
https://docs.gno.land/reference/stdlibs/std/address#isvalid

Reply:
thanks! fixed

https://github.com/gnoswap-labs/governance/pull/14/commits/348d47435f21d5b7150bfee0d5bf56089a7eb12c